DeviceTests/DeviceTests.iOS/Info.plist

```diff
@@ -55,5 +55,9 @@
 	<string>1.0.1.0</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Access to your location is required for cool things to happen!</string>
+  <key>NSCalendarsUsageDescription</key>
+  <string>Access to view and create calendar events</string>
+  <key>NSRemindersUsageDescription</key>
```

Comment:
Can we remove NSRemindersUsageDescription and confirm whether we can still read the calendar events

Reply:
can be removed addressed in: https://github.com/builttoroam/Essentials/pull/43